modules/tensor_mechanics/doc/content/source/materials/ADLAROMCreepStressUpdate.md

```diff
@@ -0,0 +1,168 @@
+
+As a first approximation, each component of the stress field can be described by a Gaussian
+function. Therefore, the probability distribution function $P\left(\tau^s-{\bar{\tau}}^s\right)$,
+representing the volume fraction of sub-material points with $\tau^s$, can be written as:
+\begin{equation}
+  P\left(\tau^s-{\bar{\tau}}^s\right)=\ \frac{1}{\sqrt{2\pi V}}\exp{\left(-\frac{\left(\tau^s-{\bar{\tau}}^s\right)^2}{2V^2}\right)},
+\end{equation}
+where $V$ is the distribution variance. [!cite](Wang:2016fs) suggest that the dispersion of intragranular stress
+depends on the total dislocation density, and express $V$ as,
+\begin{equation}
+  V=\ \eta\sqrt{\rho_{cell}+\rho_{cw}},
+\end{equation}
+Where $\eta={10}^7$ MPa/m is an effective scaling coefficient, introduced here for the sake of simplicity. In practice, though, $\eta$ is
+expected to be a function depending on the "Contrast factor" for each dislocation population and on
+dislocation arrangements. The material parameters in use for the generation of the ROM database can
+be found elsewhere [!cite](Tallman:YBweFy2x). For the meaning and implementation of the parameters,
+readers are referred to the original work [!cite](Wen:wv).
+
+
+The polynomial regression model used as the mathematical form of the ROM is formulated in terms of orthogonal polynomial
+terms, defined using Legendre polynomials. These terms are expressed as $P_i\left(x\right)$,
+polynomial of degree $i$, evaluated for input $x$, where the range of $x$ has been normalized to the
+$\left[-1,\ 1\right]$ interval. The model form includes polynomial terms of each input parameter and all interaction terms thereof, i.e.,
+\begin{equation}
+	Output\ \sim \sum_{w=0}^{deg}{\ldots\sum_{z=0}^{deg}{\alpha_{w\ldots z}P_w\left(x_w\right)\ldots}}P_z\left(x_z\right),
+\end{equation}
+where $deg$ is the maximum degree of polynomial to be used in the model, $\alpha_{w\ldots z}$ is the regression coefficient for the term formed from the
+ product of $w$-th degree polynomial of input $x_w$, etc.
+
+Three output values are required from the ROM: $\dot{\epsilon}_{vm}$, ${\dot{\rho}}_{cell}$, and
+${\dot{\rho}}_W$. While the strain rate is the desired output for predicting creep behavior, The
+defect density rates are also needed, as they affect the evolution of the creep rate over time
+through their own evolution. An entirely independent regression model is employed to predict each of
+the three outputs required. For example, the regression model for $\dot{\epsilon}_{vm}$ is shown, i.e.,
+\begin{equation}
+  {\dot{\varepsilon}}_{vm}\ ~\ \sum_{n_\varepsilon}^{deg}\sum_{n_{\rho_{cell}}}^{deg}\sum_{n_{\rho_W}}^{deg}\sum_{n_\sigma}^{deg}\sum_{n_T=0}^{deg}{\alpha_{n_\varepsilon n_{\rho_{cell}}n_{\rho_W}n_\sigma n_T}^{{\dot{\varepsilon}}_{vm}}P_{n_\varepsilon}\left(\varepsilon_{vm}\right)P_{n_{\rho_{cell}}}\left(\rho_{cell}\right)P_{n_{\rho_W}}\left(\rho_W\right)P_{n_\sigma}\left(\sigma_{vm}\right)P_{n_T}\left(T\right)\ }
```

Comment:
the approximation sign `\~` isn't rendering in the documentation. I am glad to see that it's detonated as an approximation since an direct equals sign would be misleading